actions/lib/Excel_Reader.py

```diff
@@ -87,7 +87,7 @@ def get_variables_for_key(self, key):
         variables = {}
         col = self._variable_start_column
         row = self.get_row_for_key(key)
-        while row>=0 and col <= self._variable_end_column:
```

Comment:
"while row>= and" needs to be in there to address when a excel_key is not found.